### ISSUE TITLE
DataGenerator for Cyrillic

### ISSUE THREAD

**Issue by rtm-1dyakonov** [Belval/TextRecognitionDataGenerator#306]
Hello!
Your generators are mentioned in EasyOCR repository as an advised method for generating data for training custom text recognition models.
Is there any opportunity to generate dataset of Cyrillic symbols with your generators ?

**Comment by Deeksha-5** [comment]
Yes, it's very much possible, I tried generating for the Russian Cyrillic ones. It worked well.
